### PR TITLE
dev-python/seaborn: version bump

### PR DESCRIPTION
**Pull request by TheChymera**
Package-Manager: portage-2.3.2

### CODE REVIEW

**Comment by TheChymera** [comment]
Maintainership request [submitted](https://bugs.gentoo.org/show_bug.cgi?id=601674).

File: dev-python/seaborn/seaborn-0.7.1.ebuild [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+	sci-libs/scipy[${PYTHON_USEDEP}]
+"
+DEPEND="${RDEPEND}
+	test? (
+		dev-python/nose[${PYTHON_USEDEP}]
+	)
+"
+
+python_test() {
+	cat > matplotlibrc <<- EOF
```
Comment (mgorny) [inline dev-python/seaborn/seaborn-0.7.1.ebuild]: `|| die` here, please (after EOF).

File: dev-python/seaborn/seaborn-0.7.1.ebuild [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+	backend : Agg
+	EOF
+	virtx nosetests --verbosity=3
```
Comment (mgorny) [inline dev-python/seaborn/seaborn-0.7.1.ebuild]: I think you need to `|| die` here as well, or at least I think `virtx` doesn't die of its own.

**Comment by TheChymera** [comment]
@mgorny changes applied

**Comment by mgorny** [comment]
There's proxy-maint project missing but we'll fix that on our end. @gentoo/python, agree on co-maint?

**Comment by monsieurp** [comment]
Yes.

**Comment by monsieurp** [comment]
Also added Proxy-maintainers to metadata.xml. See 38cd476083c27f23836e81107fa0c1f3f024d913.

**Pull request closed without merging**